packages/blaze/view.js

```diff
@@ -184,6 +184,34 @@ Blaze.View.prototype.autorun = function (f, _inViewScope) {
   return c;
 };
 
+/**
+ * Just like Blaze.View#autorun, but with Meteor.subscribe instead of
+ * Tracker.autorun. Stop the subscription when the view is destroyed.
+ * @return {SubscriptionHandle} A handle to the subscription so that you can
+ * see if it is ready, or stop it manually
+ */
+Blaze.View.prototype.subscribe = function (/* arguments */) {
+  var self = this;
+
+  if (! self.isCreated) {
+    throw new Error("View#subscribe must be called from the created callback at the earliest");
```

Comment:
Are there additional limitations? Like -- a user probably shouldn't call this from the destroyed callback?

And if it's OK to call this from the rendered callback I think it would be good to test that explicitly (in the same test if needed, just add an additional subscription from a rendered callback)